ts/components/messages/MessageComponent.tsx

```diff
@@ -53,21 +61,32 @@ class MessageComponent extends React.Component<Props> {
           });
         }}
       >
-        <Left>
-          <Text leftAlign={true} alternativeBold={true}>
-            {`${serviceOrganizationName} -  ${serviceDepartmentName}`}
-          </Text>
-          <Text leftAlign={true}>{subject}</Text>
-        </Left>
-        <Right>
-          <Text formatDate={true}>
-            {convertDateToWordDistance(
-              new Date(createdAt),
-              I18n.t("messages.yesterday")
-            )}
-          </Text>
-          <Icon name="chevron-right" />
-        </Right>
+        <View padded={paymentAmount !== undefined}>
+          <Left>
+            <Text leftAlign={true} alternativeBold={true}>
+              {`${serviceOrganizationName} -  ${serviceDepartmentName}`}
+            </Text>
+            <Text leftAlign={true}>{subject}</Text>
+          </Left>
+          <Right>
+            <Text formatDate={true}>
+              {convertDateToWordDistance(
+                new Date(createdAt),
```

Comment:
we shouldn't do the conversion from `string` to `Date` here, if `createdAt` is an invalid string, this will break the app